### ISSUE TITLE
Need bitcode support

### ISSUE THREAD

**Issue by holoman0** [audiokit/AudioKit#242]
I'm new to AudioKit. Been using for a few days. I'm in XCode 7.0.1 and when I tried to run my app on my phone I got:
```
ld: '[...]/AudioKit/AudioKit/Platforms/iOS/libsndfile.framework/libsndfile' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Found this closed issue:
https://github.com/audiokit/AudioKit/issues/204

Seems important that we be able to move forward with the current dev environment. Are there plans to get bitcode supported?
Thanks.

**Comment by megastep** [comment]
We have binaries with Bitcode support on the develop branch of the project. These will be released along with the next AudioKit version (2.3), hopefully very soon. But you can get the binaries right here by checking out the develop branch.

**Comment by holoman0** [comment]
Excellent. Should have tried that. Thanks!!!!
